prisma/schema.prisma

```diff
@@ -11,23 +11,56 @@ model Site {
   id      String   @id @default(cuid())
   name    String
   types   String[]
+  lat     Float
+  lng     Float
   city    String
   country String
   radius  Int
-  User    User?    @relation(fields: [userId], references: [id])
+  user    User?    @relation(fields: [userId], references: [id])
   userId  String?
-  lat     Float
-  lng     Float
+}
+
+model Account {
+  id                 String  @id @default(cuid())
+  userId             String
+  type               String
+  provider           String
+  providerAccountId  String
+  refresh_token      String? @db.Text
```

Comment:
Do we need Account and Session in the flood_frontend database? Isn't all of this data (that we need at least) available via the session-cookie we get from KC?

Reply:
I am not 100% sure about this. My limited knowledge of KC makes it hard to think of both pros and cons of having this stored in the flood-frontend db. I would imagine it allows the user to close the app and reopen it again, while not having to re-authenticate because we have a session stored.

Reply:
I think the session is stored in the browsers localstorage or something like that. So it should still be available after reopening the browser. I am not 100% sure how Auth.js handles this though, but this should be easy to verify by reopening the browser